portals/devportal/source/src/app/components/Apis/Details/Environments.jsx

```diff
@@ -158,6 +158,15 @@ function Environments(props) {
             return null;
         }
     };
+
+    const externalEndpoint = () => {
+        if (api.advertiseInfo.apiExternalProductionEndpoint) {
```

Comment:
simplify the function

Reply:
done with https://github.com/wso2/apim-apps/pull/130/commits/b138ea609173d3c2c23316e65e3a4b109fb237f5